its/ruling/src/test/java/org/sonarsource/iac/IacRulingTest.java

```diff
@@ -104,7 +104,8 @@ void test_kubernetes() throws IOException {
   @Test
   void test_docker() throws IOException {
     Map<String, String> properties = new HashMap<>();
-    properties.put("sonar.inclusions", "sources/docker/**/Dockerfile*, ruling/src/test/resources/sources/docker/**/Dockerfile*");
+    properties.put("sonar.inclusions", "sources/docker/**/Dockerfile*, ruling/src/test/resources/sources/docker/**/Dockerfile*, " +
+      "ruling/src/test/resources/sources/docker/**/**.Dockerfile, ruling/src/test/resources/sources/docker/**/**.dockerfile");
```

Comment:
Please also add the new suffixes for the sources/docker directory. So we can add [this](https://github.com/SonarSource/public-dockerfiles) git submodule.